platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/attribution/Attribution.java

```diff
@@ -4,7 +4,7 @@
 
   private static final String OPENSTREETMAP = "OpenStreetMap";
   private static final String OPENSTREETMAP_ABBR = "OSM";
-  static final String TELEMETRY = "Telemetry Settings";
+  static final String TELEMETRY = "TelemetryBase Settings";
```

Comment:
I think this is an unnecessary result of the class name change?

Reply:
refactor issue, nice catch